src/style.css

```diff
@@ -72,3 +85,26 @@ td:nth-child(even) {
   box-shadow: 2px 2px var(--grey);
   margin-inline-start: auto;
 }
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/210593991-94f3096c-180c-4bd0-a552-426f3570af15.png)

- [ ] Your link to your license file is broken :fearful: Kindly fix this link :blush: I guess this link is not working because you don't have a license file in your project :smile: You can add a license file using this [template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) :+1:
- [ ] After adding the license file Kindly add your name, current year and app type in your MIT.md file :+1: to make your project licensed :+1:
- [ ] After making the above changes kindly verify that your link to your license file is working which is in readme :thinking: